db/schema_applier.cc

```diff
@@ -22,6 +22,8 @@
 
 #include <fmt/ranges.h>
 
+#include "seastar/core/shard_id.hh"
+#include "seastar/core/sharded.hh"
```

Comment:
Use angle brackets for including Seastar API (as it's a \<system library> from our perspective).

Reply:
Yes, finally found config for clangd for that: https://github.com/llvm/llvm-project/pull/67749